hydra-node/src/Hydra/Logging/Monitoring.hs

```diff
@@ -94,6 +94,7 @@ monitor transactionsMap metricsMap = \case
     -- transactions after some timeout expires
     atomically $ modifyTVar' transactionsMap (Map.insert (txId tx) t)
     tick "hydra_head_requested_tx"
+  -- REIVEW! should we handle SnapshotSideLoaded ???
```

Comment:
I think it would be a nice addition since it is as important as `SnapshotConfirmed` imo

Reply:
We discussed this on friday: We **must** have only one event that can be counted as a `SnapshotConfirmed`.

Additional info like a side load happened can go into another `StateEvent` / `ServerOutput`. That means, the head logic which does a side load should result in two state changes.